lib/ukdebug/gdbstub.c

```diff
@@ -135,6 +136,43 @@ static int gdb_hex2int(char hex)
 	return -EINVAL;
 }
 
+static unsigned long gdb_hex2ulong(const char *buf, __sz buf_len, char **endptr)
```

Comment:
I'm not sure how important that is.

Reply:
IMO it is, as these functions are in the scope of the `isrlib`, and we should avoid code duplication in general. The same probably applies to `strntoull()` that is defined in `libukparam`. @skuenzer thoughts?

Reply:
Hum... a good question. I remember a design choice was to have a self-contained GDB code base as much as possible to avoid to rely on potential broken libc functions and make sure debugging always works, even of those. Ideally, the GDB stub is independent of any libc (also because weird dependencies to TLS sometimes).

Reply:
Yes, I agree with this.